Hackdays (#32)
camunda-community-hub/zeebe-client-rust: README.md

# Zeebe Rust Client
Rust client for Zeebe in early stages. Suitable for early adopters

Features
* CLI for all commands based on Zeebe Client 8.0
* Support for OAuth authentication

Next Steps
1. Worker implementation
2. Publish crates
3. Build an application that uses this Rust client

## Cli Tool

> **Warning**
> The Cli Tool might leak credentials in log messages

Run `cargo run -- help` to see available commands and options.

**Authentication for Camunda Cloud**

> First, [generate and download](https://docs.camunda.io/docs/next/components/console/manage-clusters/manage-api-clients/) client credentials from Console. Let's assume they are in a file called credentials.txt.

Author: ```suggestion
First, [generate and download](https://docs.camunda.io/docs/next/components/console/manage-clusters/manage-api-clients/) client credentials for Camunda Cloud. Let's assume they are in a file called credentials.txt.
```
